src/main/kotlin/org/malachite/estella/mails/MailTexts.kt

```diff
@@ -0,0 +1,19 @@
+package org.malachite.estella.mails
+
+object MailTexts {
```

Comment:
ogólnie bym nie robił tego obiektu, tylko zostawił same funkcje - w Kotlinie je też da się normalnie importować, a nie jest to Java, gdzie musisz wszystko opakowywać w klasy / obiekty

Reply:
Ale jak nie chce Ci się już zmieniać, to może zostać